app/src/main/res/values/strings.xml

```diff
@@ -130,6 +130,10 @@
     <string name="show_answers">Show answers</string>
     <string name="failed_to_load_font">Failed to load custom font: %s</string>
     <string name="failed_to_load_resource">Failed to load resource: %s</string>
+    <string name="keep_on_sum">Screen keeps lit in Study</string>
```

Comment:
Can you rename the string to "keep_screen_on_summary"? Also, I'd write "Keep screen lit during tests".